bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -85,19 +93,41 @@ rdir = layout.SectionNumber('webpage', ['offsource_triggers_vs_time',
 _workflow.makedir(rdir.base)
 _workflow.makedir(rdir['workflow'])
 
-# Input trigger file
-trig_file = start_rundir+'/'+args.trig_file 
+# Input trigger files
+# Expected structure: [ALL_TIMES, ONSOURCE, OFFSOURCE, OFFTRIAL_1, ..., OFFTRIAL_N]
+# Check filename in case this doesn't happen
+all_times_file = None
+onsource_file = None
+offsource_file = None
+offtrial_files = []
+for trig_file in args.trig_files:
+    if not os.path.isfile(trig_file):
+        raise ValueError("Trigger file %s does not exist" % trig_file)
+    if "ALL_TIMES" in trig_file:
+        all_times_file = os.path.join(start_rundir, trig_file)
+    if "ONSOURCE" in trig_file:
+        onsource_file = os.path.join(start_rundir, trig_file)
+    if "OFFSOURCE" in trig_file:
+        offsource_file = os.path.join(start_rundir, trig_file)
+    if "OFFTRIAL" in trig_file:
+        offtrial_files.append(os.path.join(start_rundir, trig_file))
```

Comment:
My request above is to tell the user we expect this order in the FileList, but we should not rely on file names. Rather:
```
all_times_file = os.path.join(start_rundir, args.trig_files[0])
onsource_file  = os.path.join(start_rundir, args.trig_files[1])
offsource_file = os.path.join(start_rundir, args.trig_files[2])
offtrial_files = [os.path.join(start_rundir, trig_file) for trig_file in args.trig_files[3:]]
```
and at each step inform the user with messages like "using such-and-such as all-times trigger file" etc.